Description: Fix FTBFS with Boost 1.67
 In Boost 1.67 parameters passed to time constructors must be
 integral. This does not change the previos behaviour, since
 the count was already stored as an integer anyway. The only
 different is that now the conversion must be explicit.
Author: Giovanni Mascellani <gio@debian.org>
Bug-Debian: https://bugs.debian.org/914056

--- cclive-0.9.3.orig/src/cc/progressbar.h
+++ cclive-0.9.3/src/cc/progressbar.h
@@ -316,7 +316,7 @@ private:
 
   static inline std::string eta_from_seconds(const double s)
   {
-    const pt::time_duration& td = pt::seconds(s);
+    const pt::time_duration& td = pt::seconds(static_cast<long>(s));
     return pt::to_simple_string(td);
   }
 
